docs/status.md

```diff
@@ -129,7 +129,7 @@ one of the following tracking labels.
 | rng                      | yes           | yes          | yes            | yes             | no          |
 | rng_bit_generator        | no            | yes*         | infeasible     | yes             | no          |
 | round_nearest_afz        | no            | yes*         | yes*           | yes             | no          |
-| round_nearest_even       | no            | revisit      | revisit        | yes             | no          |
+| round_nearest_even       | yes           | yes          | yes            | yes             | no          |
```

Comment:
@ghpvnist did you confirm compilation failure for complex types

Reply:
Why was verification and shape inference marked `revisit` before and not `yes*`?

Reply:
It was part of a large refactoring of this file in #331, and whichever ones that had some sort of verification/type inference was set to `revisit`.